frontend/src/components/Board/Board.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from 'react';
+import {
+  BoardContainer,
+  BoardInfoContainer,
+  BoardInfo,
+  BoardImage,
+  BoardTitle,
+  AuthorInfoContainer,
+  AuthorName,
+  AuthorAvatar
+} from "./Board.styled";
+import DotsButton from "../DotsButton/DotsButton";
```

Comment:
```suggestion
import DotsButton from "components/DotsButton/DotsButton";
```
Sprawdź, czy taka (lub podobna) ścieżka zadziała.

Reply:
nie działa, uzylem tej, którą podpowiedzial mi intellij